### PR TITLE
HSEARCH-1798 EAR

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/HSEARCH-1798

EAR deployment integration tests

### CODE REVIEW

File: integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/wildfly/ModuleMemberRegistrationEarArchiveWithJbossDeploymentIT.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.descriptor.api.Descriptors;
+import org.jboss.shrinkwrap.descriptor.api.application6.ApplicationDescriptor;
+import org.jboss.shrinkwrap.descriptor.api.persistence20.PersistenceDescriptor;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Example of an integration test using JBoss AS 7 and Arquillian.
```
Comment (gunnarmorling) [inline integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/wildfly/ModuleMemberRegistrationEarArchiveWithJbossDeploymentIT.java]: Why is it an example? Also it's WF 8, right?

**Comment by gunnarmorling** [comment]
Looks good to me apart from the class-level comments on the test classes.

File: integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/wildfly/ModuleMemberRegistrationEarArchiveWithJbossDeploymentIT.java (hunk 1 above)
Comment (Sanne) [inline integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/wildfly/ModuleMemberRegistrationEarArchiveWithJbossDeploymentIT.java]: It's an example as we're using some of these tests to give advice to users on how to run such integration tests..

**Comment by Sanne** [comment]
I've merged it, as the issue pointed out by Gunnar is a more widespread problem.
Could you send a follow-up PR changing the various "JBoss 7" descriptions to WildFly, and also add a one line description on each of the various integration tests?
It's starting to get a bit hard to figure what the purpose of each IT is - the class names are a good hint but a small description would be more useful to remind what's the "special purpose" of each.

**Pull request closed without merging**